File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/identity/services/TokenService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** Domain service issuing and verifying user tokens. */
public class TokenService {
    /** Issue a token for the user described by the partial input. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Verify a token for the given user id. Returns null when not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} active token-bearing users. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Revoke the token for the given user id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/identity/services/TokenService.ts
import { User } from '../aggregates/User';

/** TokenService service. */
export class TokenService {
  create(input: Partial<User>): User { throw new Error('not implemented'); }
  findById(id: string): User | null { throw new Error('not implemented'); }
  list(limit: number): User[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}
